artman/tasks/gapic_tasks.py

```diff
@@ -126,7 +126,7 @@ def execute(self, language, toolkit_path, descriptor_set, service_yaml,
                         for yaml in service_yaml]
         args = [
             '--descriptor_set=' + os.path.abspath(descriptor_set),
-            '--package_yaml=' + os.path.abspath(package_metadata_yaml),
+            '--package_yaml2=' + os.path.abspath(package_metadata_yaml),
```

Comment:
How does this affect running artman on existing APIs? Is there any googleapis update required?

Reply:
This file is generated purely within the scope of executing artman - you can almost view the file as an object that is being passed from artman to toolkit, except that it is being passed through a file. Nothing needs to change in googleapis.